.ci/variables.json

```diff
@@ -2,6 +2,8 @@
   "python_versions": {
     "PY38": "3.8.12",
```

Comment:
Do we need 3.8 here?

Reply:
I put Python 3.8 back into the manual integration tests despite what I discussed with Govind above and mentioned in the PR description. I plan to create a separate issue to continue the discussion on how the community wishes to run integration tests on python versions